x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -47,6 +47,10 @@ export interface ComplianceDashboardData {
   trend: PostureTrend[];
 }
 
+export interface Info {
```

Comment:
This name is pretty generic, and also where it lies is not very informative (`common/types`). I suggest giving a more descriptive name